### DISCUSSION TITLE
Anzeige von "Jetzt Geöffnet" oder "Geschlossen"

### DISCUSSION THREAD

**Discussion by dacu86** [Tastaturberuf/anyStores#97] [Feature Requests / Funktionswünsche]
Hallo,

ich möchte gerne über den angezeigten Öffnungszeiten eine Info mit "Jetzt Geöffnet" und "Geschlossen" hinzufügen. Ist das machbar und wenn wie am besten? Ich komme gerade absolut nicht weiter.

**Comment by Tastaturberuf** [comment]
Das geht übers Template mit etwas PHP-Code. Allerdings ist das kein Issue. 

Als Denkanstoß:
- Aktuellen Tag per `date('D')` holen.
- In einer Schleife mit den Öffnungszeiten vergleichen.
- Aktuelle Zeit holen und vergleichen
- Ausgabe geschlossen oder offen

**Comment by Tastaturberuf** [comment]
Lege ich bei Gelegenheit als Variable an.

**Comment by Tastaturberuf** [comment]
Hinweis für mich:

```
(bool) isOpen
(array) todaysOpeningtimes
```
